js/src/oai.ts

```diff
@@ -144,23 +150,18 @@ function wrapBetaChatCompletion<
   C extends StreamingChatResponse
 >(completion: (params: P) => C): (params: P) => Promise<any> {
   return (allParams: P & SpanInfo) => {
-    const { span_info, ...params } = allParams;
-    const { messages, ...rest } = params;
-    const span = startSpan({
-      name: "OpenAI Chat Completion",
-      spanAttributes: {
-        type: SpanTypeAttribute.LLM,
-      },
-      event: mergeDicts(
+    const { span_info: _, ...params } = allParams;
+    const span = startSpan(
+      mergeDicts(
         {
-          input: messages,
-          metadata: {
-            ...rest,
+          name: "OpenAI Chat Completion",
```

Comment:
should we use this opportunity to rename this to `Chat Completion` (esp since ppl are using it w/ multiple models that are proxied)

Reply:
Done. I left it as `OpenAI Embedding` but I can change that too if people are using different models there, too.